src/test/java/org/opensearch/knn/jni/JNIServiceTests.java

```diff
@@ -66,6 +66,17 @@ public class JNIServiceTests extends KNNTestCase {
     private String faissMethod = "HNSW32,Flat";
     private String faissBinaryMethod = "BHNSW32";
 
+    void createIndex(int[] ids, long address, int dimension, String name, Map<String, Object> parameters, KNNEngine engine) {
```

Comment:
This is good , but let's move this function to end of the all test classes.

Reply:
Done